sotodlib/wiregrid/calibrate.py

```diff
@@ -0,0 +1,404 @@
+#!/usr/bin/python3
+
+# import modules
+# python standard
+import os
+import sys
+import scipy
+from scipy import odr
+import numpy as np
+from iminuit import cost, Minuit
+from matplotlib.patches import Ellipse, Circle
+
+# SO specific
+from sotodlib import core
+from sotodlib.core import Context
+from sotodlib.io import hk_utils
+import sotodlib.io.load_smurf as ls
+from sotodlib.io.load_smurf import Observations
+
+wg_degpercount = 360/52000
+stream_ids = ['ufm_mv19', 'ufm_mv18', 'ufm_mv22', 'ufm_mv29', 'ufm_mv7', 'ufm_mv9', 'ufm_mv15'] # e.g. MF1
```

Comment:
These should not be defined here. If you need this info (although I don't see it used so I think it might be a leftover from your testing) get it from the axis manager or metadata.

Reply:
Yeah, I have a question about that. What kind of style should we use when we call these wafers? I mean which notation should we use in the pipeline, ufm_mv19 or ws0?

Reply:
Preference is for wafer_slot indexing as this is purely geometric and won't change between platforms or when we swap UFMs.